specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/serviceEndpointPolicy.json

```diff
@@ -0,0 +1,634 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting SecurityRule resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyDefinition"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get network security rule in network security group": { "$ref": "./examples/ServiceEndpointPolicyDefinitionGet.json" }
+        }
+      },
+      "put": {
+        "tags": [
+          "SecurityRules"
+        ],
+        "operationId": "ServiceEndpointPolicyDefinitions_CreateOrUpdate",
+        "description": "Creates or updates a security rule in the specified service endpoint policy.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "serviceEndpointPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the service endpoint policy."
+          },
+          {
+            "name": "serviceEndpointPolicyDefinitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the service endpoint policy definition name."
+          },
+          {
+            "name": "ServiceEndpointPolicyDefinitionParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyDefinition"
+            },
+            "description": "Parameters supplied to the create or update service endpoint policy operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting SecurityRule resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyDefinition"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting SecurityRule resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyDefinition"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create security rule": { "$ref": "./examples/ServiceEndpointPolicyDefinitionCreate.json" }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/serviceEndpointPolicies/{ServiceEndpointPolicyName}/serviceEndpointPolicyDefinitions": {
```

Comment:
{ServiceEndpointPolicyName} should be {serviceEndpointPolicyName}

Reply:
done